qutebrowser/config/configdata.yml

```diff
@@ -869,7 +869,7 @@ hints.next_regexes:
     valtype:
       name: Regex
       flags: IGNORECASE
-  desc: A comma-separated list of regexes to use for 'next' links.
+  desc: A comma-separated list of regular expressions to use for 'next' links.
```

Comment:
Not sure about this, but dropping the 'A' might be a little more consistent.

"Comma-seperated list of..."

I don't mind if you think it's wrong though...